### DISCUSSION TITLE
[ARCHIVED] SemVer для методологии

### DISCUSSION THREAD

**Discussion by azinit** [feature-sliced/documentation#38] [Archive]
> Кстати а уже были идеи что методоллогия может тоже разиваться по semver? Таким обраом у нас уже есть версия 1.0. А сейчас идут обсуждения 2.0 так как явно есть breaking changes

**Comment by azinit** [comment]
Вот такое нашел

> - Хочется четкое версионирование (когда кто-то из новых разработчиков сядет за проект со старой методологией)
>   - С документацией для каждой из версий

**Reply by azinit** [reply]
- Т.е. у нас наверняка методология в текущем виде будет для `2.*` скорее всего
- Также у нас будет выделяться главная основная методология
- Но для каждой мажорной версии (как минимум) скорее всего будет держаться своя версия (@sergeysova [уже делал примерно такое](https://github.com/feature-sliced/featureslices.dev))
   -  Чтобы разрабы, которые еще не успели обновиться - могли иметь доступ к предыдущим версиям (@karina-drummer вроде была инициатором этого, и в целом звучит здраво)

**Reply by sergeysova** [reply]
хотелось бы, чтобы мажорные версии выходили “раз в десять лет”
чтобы не ломать людям жизнь

**Comment by azinit** [comment]
Вот хороший пример, как это сделано у conventionalcommits
> https://www.conventionalcommits.org/en/v1.0.0/

![image](https://user-images.githubusercontent.com/42924400/111059816-64c67800-84a9-11eb-8bdb-3f0ab8dac45b.png)
